.github/workflows/continuous_integration.yml

```diff
@@ -57,6 +58,13 @@ jobs:
         pip install git+https://github.com/goodchemistryco/Tangelo.git@develop
       if: always()
 
+    - name: Download required data to run the MIFNO notebook
```

Comment:
Pros VS cons of putting this here, in the environment setup for tests (away from the users) VS in the notebook itself ?

My fear is that environment installation would become super complicated because of data stuff across many different notebook and abstracted from user, who them won't know where the data comes from while looking at the notebook itself

Reply:
Should we just export -everything- data from the Tangelo repo here, so that all notebooks can just grab whatever they need ?